packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Columns.js

```diff
@@ -74,11 +74,6 @@ const Columns = ({
             }
           }}
           tabIndex={0}
-          onFocus={(e) => {
-            if (e.target === e.currentTarget) {
-              setFocusIndex(0);
-            }
-          }}
```

Comment:
I believe this was intended for [this behavior](https://pages.github.ibm.com/cdai-design/pal/components/data-table/column-customization/usage/#accessibility) listed in the PAL guidance.